docs/concepts/pdp/factdb.mdx

```diff
@@ -0,0 +1,55 @@
+---
+sidebar_position: 11
+title: Handling Volume ( EAP )
+---
+
+# FactDB ( EAP )
+
+FactDB is a new feature that allows you to store and query facts more efficiently in the PDP.
+It uses embedded DB technology to store facts in a more efficient way instead of the policy engine memory,
+and provides a RestAPI to query those facts.
+
+:::info EAP
+This feature is in EAP stage - please contact with us if you want to try it out.
+:::
+:::warning
+To use this feature all your PDPs must be upgraded to version 0.7.0 or above.
+Otherwise, some features might not work as expected.
+:::
+
+## Enabling FactDB
+
+To enable the FactDB on your environment there are two configurations that you need to set:
```

Comment:
3, update pdp, setttings and env in the PDP
I would write headlines and only then have the instructions